src/hooks/useCirclePost.ts

```diff
@@ -0,0 +1,139 @@
+import { useCallback } from 'react';
+import { useQuery } from 'react-query';
+import { useActiveAccount } from './useActiveAccount';
+import { gql } from 'graphql-request';
+import { useGraphQLClient } from './useGraphQLClient';
+
+export const useCirclePost = (postId: string, disabled?: boolean) => {
```

Comment:
Nit: My instinct says that this should just be `usePost` since the post is not necessarily the child of a circle and that the query itself is the `post` query.